services/app/src/components/chessBoard/Tile.jsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-underscore-dangle */
+import React, { useCallback } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+const Tile = ({
+  tile: { attribute, items, isFilled },
+  onClick,
+  row,
+  column,
+  boardLength,
+}) => {
+  const isTileWithBorder = () =>
+    !attribute && row !== boardLength - 1 && column !== 0;
+
+  const _onClick = useCallback(() => {
+    onClick(row, column);
+  }, [onClick, row, column]);
+
+  return (
+    <td
+      onClick={_onClick}
+      role="presentation"
+      className={classNames({
+        'black-tile': isFilled,
+        'white-tile': isTileWithBorder(),
```

Comment:
Naming is a bit confusing - the class name you are giving for the tile is `white-tile` but the method name says `isTileWithBorder`. Can we align those two?